dance/pipeline.py

```diff
@@ -795,4 +797,36 @@ def wandb_sweep_agent(
         logger.info(f"Spawning agent: {sweep_id=}, {entity=}, {project=}, {count=}")
         wandb.agent(sweep_id, function=function, entity=entity, project=project, count=count)
 
+        if summary_file_path is not None:
+            sweep = wandb.Api().sweep(f"{entity}/{project}/{sweep_id}")
+            summary_data = []
+            for run in sweep.runs:
+                result = dict(run.summary._json_dict).copy()
+                result.update(run.config)
+                result.update({"id": run.id})
+                summary_data.append(flatten_dict(result))  #get result and config
+            pd.DataFrame(summary_data).set_index(["id"]).to_csv(summary_file_path)  #save file
         return entity, project, sweep_id
+
+
+def flatten_dict(d, parent_key='', sep='_'):
```

Comment:
Merge the results such as acc with the relevant parameter configuration and save it to a csv file